pytest_girder/pytest_girder/utils.py

```diff
@@ -22,16 +22,6 @@
 class MockSmtpServer(smtpd.SMTPServer):
     mailQueue = queue.Queue()
 
-    def __init__(self, localaddr, remoteaddr, decode_data=False):
-        kwargs = {}
-        if six.PY3:
-            # Python 3.5+ prints a warning if 'decode_data' isn't explicitly
```

Comment:
This warning is only on exactly Python 3.5.

For 3.6+, there is no longer a need to wrap `__init__`.